### ISSUE TITLE
why bjiyxo trained weight stronger than gcp

### ISSUE THREAD

**Issue by l1t1** [gcp/leela-zero#2106]
recent promoted 3 weights are all bjiyxos
and his weights in match vs 196 stronger in average
of the training games are same,then his training parameters are better, if it is true,should we use his parameter in traning?

**Comment by herazul** [comment]
Main difference is bjixyo kept some ELF v1 games in the training.
Difficult to know why it helps, there has been some discussion about maybe m999 being to harsh for improving late game skills, butwe have no evidence whatsoever.
Also keep in mind that it's only three net, it could also have been luck. (some official net almost promoted too.)

**Comment by Ishinoshita** [comment]
IIRC, @bjiyxo training weights 4:1 policy loss vs value loss and kept 25% of Elf1 games.

Elf1 games differ from LZ games both by being played by Elf1...;-), and by being played only with resign enabled. No long tails of past resign threshold positions in the training data, where search policy completely flattens and Dirichlet noise is much emphasized. Elf1 games were played with -m 999 as well, which randomizes game outcome and thus value target much more than -m 30, but this is minored by the loss weighting. All in all, @bjixyo net training is less sensitive to noise injected into value target by -m 999 setting.

But in the long run, is it desirable de-emphasize value vs policy ? AG breakthrough was about acheiving to train a value network... Still so many things unsorted here !

**Comment by bjiyxo** [comment]
We had a hard time before weight 194, so I'm not sure our parameter is better. We shall wait a while to come to a conclusion.

**Comment by l1t1** [comment]
all gcps weight played the match, but bjiyxos weight not, maybe he test them first, only let stronger weights play matchs

**Comment by l1t1** [comment]
gcps weight is passed now. may be bjiyxos passed 3 times only by lucky is right.

**Comment by dbosst** [comment]
[dumbpass resign match game in the last match promotion set](http://zero.sjeng.org/viewmatch/636e45ecaeafd5602d92db265f0a6ba4d063c73f577b76b29e0ae96078dd7742?viewer=wgo)

A little off topic, but rather than starting a issue, does having match games like this where the engine passes instead of resigning in the middle game... doesn't a game like this with passes mess up the training.  If I'm the engine and above the 5% resign I'm always going to consider long shot tesuji like moves rather than passes (at the very least in timed matches it would be extremely advantageous, as opposed to the opposite of using up your thinking time and then passing....)

**Comment by Ishinoshita** [comment]
@dbosst 
>Does LZ use the score result number as a training feature? Or does it only consider win/loss?

For sure, LZ training data contain *nothing* related to the score result, only the game outcome. See [Training data format](https://github.com/gcp/leela-zero#training-data-format).

>doesn't a game like this with passes mess up the training?

Match games are not used for training. They represent a negligible fraction of the games sent by the server.

**Comment by john45678** [comment]
A bit off topic as well, I'd like to wish gcp and all here a Happy New Year, So much has been done with LZ in just over the year, this time last year we were debating if LZ could beat Gnugo :)

**Comment by l1t1** [comment]
198 lost to 197 as 0/4  http://www.yss-aya.com/cgos/19x19/cross/LZ_197_56be_p400.html

**Comment by herazul** [comment]
@l1t1 and ?

**Comment by l1t1** [comment]
198 seems not as good as its elo shows. it is very lucky.  https://github.com/gcp/leela-zero/issues/2116

**Comment by Glrr** [comment]
No need to worry. If 198 is weak, it will be quickly overthrown.

**Comment by gcp** [comment]
I suspect we don't use the same learning rate either? I suspect my learning rate right now is on the high end, but thanks to SWA there's no pressing need to drop, as networks are still promoting.

**Comment by gcp** [comment]
0.0001 @ BS 96
Learning rate is meaningless without knowing batch size!

**Comment by bjiyxo** [comment]
lr=0.0001(1e-4) @ BS 240
